swagger.yaml

```
@@ -161,6 +161,34 @@ info:
         </code>
       </p>
 
+    <br><br>
+    <h3>Partial update</h3>
+      <table class="table table-striped table-bordered">
+        <thead>
+          <tr>
+            <td>Name</td>
+            <td>Type</td>
+            <td>Description</td>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>partial</td>
+            <td>boolean</td>
+            <td>Whether you would like to provide a partial update or not.</td>
+          </tr>
+        </tbody>
+      </table>
+      <p>
+        Using the partial field, you can provide only the updated fields in the request body while updating a resource instead of sending all the fields:
+      </p>
+      <p class="url">
+        <code class="bluetext">
+          <span class="get-text">PUT</span>
```

Assuming this should be `put-text` rather than `get-text`
```suggestion
          <span class="put-text">PUT</span>
```